src/UNL/UCBCN/Calendars.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace UNL\UCBCN;
+
+use UNL\UCBCN\ActiveRecord\RecordList;
+use UNL\UCBCN\ActiveRecord\Record;
+/**
+ * Object related to a list of calendars.
+ * 
+ * PHP version 5
+ * 
+ * @category  Events 
+ * @package   UNL_UCBCN
+ * @author    Tyler Lemburg <trlemburg@gmail.com>
+ * @copyright 2015 Regents of the University of Nebraska
+ * @license   http://www1.unl.edu/wdn/wiki/Software_License BSD License
+ * @link      http://code.google.com/p/unl-event-publisher/
+ */
+
+class Calendars extends RecordList
+{
+    public function getDefaultOptions() {
+        return array(
+            'listClass' =>  __CLASS__,
+            'itemClass' => __NAMESPACE__ . '\\Calendar',
+        );
+    }
+
+    public function getSQL() {
+        if (array_key_exists('account_id', $this->options)) {
```

Comment:
Well the difference is that these ARE two different functions. `isset`, while used by many, is not actually meant to check if the array key is there; it checks if the variable exists and is not NULL. Well, I don't want to check if it's NULL or not here, I just want to see if account_id is in options. I don't think speed is going to be a factor on this either; the stackoverflow says they are both O(1).